### PR TITLE
[FEATURE] Subtask - (Enum) algos & evaluator

### PR DESCRIPTION
**Pull request by MatthewK3023**

### CODE REVIEW

**Review by NoctisHong — CHANGES_REQUESTED** [review]
Make decanter-ai-core-sdk enums under a enum package to maintain.
Like `DecanterAIEnums`.

File: examples/example.py [hunk 1]
```diff
@@ -70,11 +70,11 @@ def main():
     # Settings for training model using TrainInput.
     train_input = TrainInput(
         data=train_data, target='Survived',
-        algos=[Alg.XGBoost], max_model=2, tolerance=0.9)
+        algos=[Alg.XGBoost.value], max_model=2, tolerance=0.9)
```
Comment (schwannden) [inline examples/example.py:73]: .value should only appear in API request

File: examples/auto_time_series_example.py [hunk 2]
```diff
@@ -39,11 +41,11 @@ def main():
     # Settings for time series forecast training.
     train_input = TrainTSInput(
         data=train_data, target='regression', forecast_horizon=7, gap=0,
-        datetime_column='date', max_model=1, evaluator='r2', time_unit='day',
+        datetime_column='date', max_model=1, evaluator=Eva.r2.value, time_unit='day',
```
Comment (schwannden) [inline examples/auto_time_series_example.py:44]: still has value here

File: src/decanter/core/enums/__init__.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+"""Init jobs package"""
+from .evaluators import Evaluator
+from .algorithms import Algos
+
+def check_is_enum(Enum, enum):
```
Comment (schwannden) [inline src/decanter/core/enums/__init__.py:5]: this is not quite how we usually check for enum, but it is ok for now, I'll note this down, we can refactor in some future

File: src/decanter/core/enums/algorithms.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
+"""
+Function for user access the Algorithms
+of the Decanter AI Core SDK.
+"""
+from enum import Enum
+
+class Algos(Enum):
+    """
+    Evaluator are the algorithms currently
```
Comment (NoctisHong) [inline src/decanter/core/enums/algorithms.py:9]: Algos is the

File: src/decanter/core/enums/evaluators.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
+"""
+Function for user access the Metrics 
+of the Decanter AI Core SDK.
+"""
+from enum import Enum
+
+class Evaluator(Enum):
+    """
+    Evaluator are the metrics currently
```
Comment (NoctisHong) [inline src/decanter/core/enums/evaluators.py:9]: Evaluator is

**Review by NoctisHong — APPROVED** [review]

File: examples/example.py (hunk 1 above)
Comment (MatthewK3023) [reply examples/example.py]: done

File: examples/auto_time_series_example.py (hunk 2 above)
Comment (MatthewK3023) [reply examples/auto_time_series_example.py]: done

File: src/decanter/core/enums/__init__.py (hunk 3 above)
Comment (MatthewK3023) [reply src/decanter/core/enums/__init__.py:5]: ok

File: src/decanter/core/algorithms/algorithms.py [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+An enumeration is a set of symbolic names (members)
+bound to unique, constant values. Within an enumeration,
+the members can be compared by identity, and the enumeration
+itself can be iterated over.
+"""
+from enum import Enum
+
+class Algos(Enum):
+    """
+    The class Algos are the machined learning algorithms
```
[the remark this answers is not in the document]
Comment (MatthewK3023) [reply src/decanter/core/algorithms/algorithms.py]: done

File: src/decanter/core/algorithms/algorithms.py [hunk 7]
```diff
@@ -0,0 +1,19 @@
+"""
+An enumeration is a set of symbolic names (members)
+bound to unique, constant values. Within an enumeration,
+the members can be compared by identity, and the enumeration
+itself can be iterated over.
+"""
```
[the remark this answers is not in the document]
Comment (MatthewK3023) [reply src/decanter/core/algorithms/algorithms.py]: done

File: src/decanter/core/evaluators/evaluators.py [hunk 8]
```diff
@@ -0,0 +1,25 @@
+"""
+An enumeration is a set of symbolic names (members)
+bound to unique, constant values. Within an enumeration,
+the members can be compared by identity, and the enumeration
+itself can be iterated over.
+"""
```
[the remark this answers is not in the document]
Comment (MatthewK3023) [reply src/decanter/core/evaluators/evaluators.py]: done

File: src/decanter/core/evaluators/evaluators.py [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+from enum import Enum
+
+class Evaluator(Enum):
+    """
+    the class Evaluator are the metrics
```
[the remark this answers is not in the document]
Comment (MatthewK3023) [reply src/decanter/core/evaluators/evaluators.py]: done

File: src/decanter/core/enums/algorithms.py (hunk 4 above)
Comment (MatthewK3023) [reply src/decanter/core/enums/algorithms.py]: done

File: src/decanter/core/enums/evaluators.py (hunk 5 above)
Comment (MatthewK3023) [reply src/decanter/core/enums/evaluators.py]: done

**Review by NoctisHong — APPROVED** [review]

File: src/decanter/core/client.py [hunk 10]
```diff
@@ -45,6 +47,11 @@ def __init__(self, username, password, host):
         Context.create(username=username, password=password, host=host)
         
 
+    @staticmethod
+    def create(
+            username, password, host):
+            Context.create(username=username, password=password, host=host)
+
```
[the remark this answers is not in the document]
Comment (MatthewK3023) [reply src/decanter/core/client.py]: done

File: src/decanter/core/core_api/train_input.py [hunk 11]
```diff
@@ -24,7 +26,7 @@ class TrainInput:
         .. code-block:: python
 
             train_input = TrainInput(data=train_data, target='Survived',
-            algos=["XGBoost"], max_model=2, tolerance=0.9)
+            algo=Alg.XGBoost, max_model=2, tolerance=0.9)
```
[the remark this answers is not in the document]
Comment (MatthewK3023) [reply src/decanter/core/core_api/train_input.py]: done

**Pull request merged**